.github/workflows/ci.yml

```
@@ -81,7 +81,7 @@ jobs:
       - name: Build docs using sphinx
         run: |
           cd docs
-          poetry run sphinx-build -W --keep-going source build
+          poetry run sphinx-build --jobs auto -W --keep-going source build
```

Would it also work if you just ran `make` here? Or will it not run without the `poetry run` prefix?

---

I think `poetry run make` would work - but I think we probably do want to run the local `sphinx-build` in a slightly different way to the CI? So fail fast and not keep going locally but find everything wrong on CI? Doesn't make a huge difference but I'll leave as is for now.